application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -191,7 +187,8 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
                     getKeyByValue(VC_APPLICATION_TO_ID, applicationId).orElse("an activity");
         }
 
-        handleSubcommand(event, voiceChannel, applicationId, maxUses, maxAge, applicationName);
+        handleSubcommand(event, voiceChannel, applicationId, maxUses.getAsInt(), maxAge.getAsInt(),
```

Comment:
And what if the optional is not present? Then this `.getAsInt()` call crashes! You clearly havent tested this. This is a critical bug.